### PR TITLE
Raise Error when the file is not valid

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38348

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by roysagi — COMMENTED** [review]
@ChanochShayner - this looks good let's just make sure we check every possible flow:
[x] Regular content build:
     With a file with the file name change.
     With a newly added file.
     With a file modified.
[x] SDK nightly.
[x] Private build.
[x] Upload flow

Some of those you've already done - just check the boxes.

File: demisto_sdk/commands/common/tests/tools_test.py [hunk 1]
```diff
@@ -74,7 +75,9 @@ def test_get_file(self, file_path, func):
 
     def test_get_file_exception(self):
```
[the remark this answers is not in the document]
Comment (ChanochShayner) [reply demisto_sdk/commands/common/tests/tools_test.py:68]: done

**Comment by ChanochShayner** [comment]
> @ChanochShayner - this looks good let's just make sure we check every possible flow:
> [x] Regular content build:
> With a file with the file name change.
> With a newly added file.
> With a file modified.
> [x] SDK nightly.
> [x] Private build.
> [x] Upload flow
> 
> Some of those you've already done - just check the boxes.

@roysagi All tests passed well, ready to merge.

**Review by roysagi — APPROVED** [review]
Please add a changelog entry - good job!

**Pull request merged**